components/Common/Banner/index.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from 'next/link';
+import { ArrowUpRightIcon } from '@heroicons/react/24/outline';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type BannerProps = {
+  type: 'default' | 'error' | 'warning';
+  text: string;
+  url?: string;
+};
+
+const Banner: FC<BannerProps> = ({ type, text, url }) => (
+  <div className={`${styles.banner} ${styles[type]}`}>
```

Comment:
```suggestion
  <div className={`${styles.banner} ${styles[type] || 'default'}`}>
```

Reply:
Shouldn't it be that 
```suggestion
  <div className={`${styles.banner} ${styles[type || 'default']}`}>
```

Reply:
Well that's not my intent here.

The issue we have is that if there is no style with the provided type, then the banner will have no style.

Either we always apply the default style if no style provided and/or invalid styles provided.

I know TypeScript in theory should prevent this from happening at least on a developer perspective,  but knowing that this component will be used dynamically, added safety is always a good thing.

Reply:
> added safety is always a good thing.

I'm with you 100%. I will read the docs for all the possibilities shown above and find the most suitable one.

Reply:
I had update take a look an tell me what do you think